src/components/calculator.js

```diff
@@ -6,10 +6,15 @@ const Calculator = () => {
   const [data, setData] = useState({ total: 0, next: '', operation: '' });
 
   // creat handler for button in sfc
-  const HandleClick = (element) => {
+  // const HandleClick = (element) => {
+  //   const math = calculate(data, element.target.textContent);
+  //   setData({ ...data, ...math });
+  // };
+
+  function HandleClick(element) {
```

Comment:
- [ ] You have used a descriptive function name. However, function names should be written in camelcase to better use the [Javascript best practices](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices).